core/src/main/java/org/apache/calcite/rel/core/AggregateCall.java

```diff
@@ -129,58 +111,8 @@ private AggregateCall(SqlAggFunction aggFunction, boolean distinct,
 
   //~ Methods ----------------------------------------------------------------
 
-  @Deprecated // to be removed before 2.0
-  public static AggregateCall create(SqlAggFunction aggFunction,
-      boolean distinct, List<Integer> argList, int groupCount, RelNode input,
-      @Nullable RelDataType type, @Nullable String name) {
-    return create(aggFunction, distinct, false, false,
-        ImmutableList.of(), argList, -1,
-        null, RelCollations.EMPTY, groupCount, input, type, name);
-  }
-
-  @Deprecated // to be removed before 2.0
-  public static AggregateCall create(SqlAggFunction aggFunction,
-      boolean distinct, List<Integer> argList, int filterArg, int groupCount,
-      RelNode input, @Nullable RelDataType type, @Nullable String name) {
-    return create(aggFunction, distinct, false, false,
-        ImmutableList.of(), argList, filterArg,
-        null, RelCollations.EMPTY, groupCount, input, type, name);
-  }
-
-  @Deprecated // to be removed before 2.0
-  public static AggregateCall create(SqlAggFunction aggFunction,
-      boolean distinct, boolean approximate, List<Integer> argList,
-      int filterArg, int groupCount,
-      RelNode input, @Nullable RelDataType type, @Nullable String name) {
-    return create(aggFunction, distinct, approximate, false,
-        ImmutableList.of(), argList,
-        filterArg, null, RelCollations.EMPTY, groupCount, input, type, name);
-  }
-
-  @Deprecated // to be removed before 2.0
-  public static AggregateCall create(SqlAggFunction aggFunction,
-      boolean distinct, boolean approximate, List<Integer> argList,
-      int filterArg, RelCollation collation, int groupCount,
-      RelNode input, @Nullable RelDataType type, @Nullable String name) {
-    return create(aggFunction, distinct, approximate, false,
-        ImmutableList.of(), argList, filterArg,
-        null, collation, groupCount, input, type, name);
-  }
-
-  @Deprecated // to be removed before 2.0
-  public static AggregateCall create(SqlAggFunction aggFunction,
-      boolean distinct, boolean approximate, boolean ignoreNulls,
-      List<Integer> argList, int filterArg,
-      @Nullable ImmutableBitSet distinctKeys, RelCollation collation,
-      int groupCount,
-      RelNode input, @Nullable RelDataType type, @Nullable String name) {
-    return create(aggFunction, distinct, approximate, ignoreNulls,
-        ImmutableList.of(), argList, filterArg,
-        distinctKeys, collation, groupCount, input, type, name);
-  }
-
```

Comment:
Sorry, we can't remove deprecated stuff.

Reply:
I have put all the deprecated functions back